### ISSUE TITLE
pdflatex not properly configured

### ISSUE THREAD

**Issue by TimFH** [TeXworks/texworks#944]
I just installed the "stable" flatpak version of texworks, but when I run it with flatpak run org.tug.texworks" the menu for selecting which tex version to use has everything from "LuaLaTex" to "MakeIndex" grayed out, and when I try to run the "pdfLaTex" command that seemed to be the only one available on a simple test file, it didn't run but a message appeared on the lower left which said "pdflatex not properly configured."

I guess this means texworks is not finding my TeX installation, but I know it's there because I can type things like LuaLaTeX and that program comes up from /usr/bin, which is listed in "Preferences". Furthermore I've been using another version of texworks for quite some time, although I recently noticed that lualatexotfload-main was not working and it's been falling back on OT1, so I decided to try the flatpak version. Hence this problem. It would also be nice if it were possible to open ".tex" files by double-clicking them, as I was able to do with my other version.

In short, how do I complete the flatpak setup?

**Comment by rinmalavi** [comment]
Same issue, 
no documentation on how to configure,
no clear way to get more logs.

**Comment by stloeffler** [comment]
Flatpak uses a sandboxed environment, i.e., it (generally) can't execute arbitrary binaries (from "outside" the flatpak world). This is a [known issue](https://github.com/flathub/org.tug.texworks/issues/11). To work around it, you can install TeX as a flatpak extension via `flatpak install org.freedesktop.Sdk.Extension.texlive//20.08` (see <https://flathub.org/apps/details/org.tug.texworks>).

**Comment by ymir-twice** [comment]
I have the same questions and now have solved it. Just add four folders into Path.
